doc/gitian-building.md

```diff
@@ -124,4 +124,67 @@ Gitian build.
 
  `./gitian-build.py --detach-sign -s $NAME $VERSION --nocommit`
 
-Make another pull request for these.
\ No newline at end of file
+Make another pull request for these.
+
+Simplified Guide
+-----------------------------
+
+Prerequesites: 
+ 1) have a Github Account
+ 2) have a known PGP key, like on keybase
+ 3) be running ubuntu or similar
+ 4) have some command line experience
+
+Step 1: 
+Go to https://github.com/dashpay/gitian.sigs and hit "Fork" in the upper left, and wait for github to finish forking the repository
+
+Step 2:
+Open a command line and create a new directory to house all of the gitian building and then cd into it. This can be
+wherever you'd like but I placed it in my home directory, something like:
+```
+cd ~
+mkdir gitian
+cd gitian
+```
+
+enter below, where "githubusername" is your github username and "0.16.0.0" is the version you are building for
+```
+export NAME=githubusername
+export VERSION=0.16.0.0
+```
+
+Then, you need to get the gitian-build.py script.
+```
+wget https://raw.githubusercontent.com/dashpay/dash/v${VERSION}/contrib/gitian-build.py
+chmod +x gitian-build.py
+```
+
+At this point if you don't have python3 and git installed, install it
+```
+sudo apt install python3 git
+```
+
+Clone your gitian.sigs repo
+```
+git clone https://github.com/$NAME/gitian.sigs
+```
+
+Now, run gitian-build.py script in setup mode. You may get asked to enter your password. 
+```
+./gitian-build.py -V lxc --setup $NAME $VERSION
```

Comment:
Why specify `-V lxc`? That's only necessary if someone doesn't have Docker, right?

Reply:
It's how I've always done it on ubuntu